Fixed Drying Rack

WewLads opened the pull request:
Fixed the drying rack. The drying rack can now properly dry anything labeled as a consumable plant.

:cl: Fixed Drying Rack
fix: Fixed the Drying rack refusing plants.
/:cl:

[why]: We need to smoke damn it!

WewLads committed:
You can now slap butts

Disarm Intent + Target Groin = Butt Slap. Deals 1 damage to both the
victim's torso and the target's active hand.

WewLads committed:
Fixed the drying rack

WewLads committed:
Revert "You can now slap butts"

This reverts commit 9b1d6808d8e24cc9b2f5d28384b3ffe633d811fe.

deathride58 commented on code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:
```diff
@@ -278,6 +278,8 @@
 
 /obj/machinery/smartfridge/drying_rack/accept_check(obj/item/O)
+	if(istype(O, /obj/item/reagent_containers/food/snacks/grown))
+		return TRUE
```
This will cause runtimes when putting things that normally don't go inside the drying rack in, since this completely bypasses the check for an item's dried_type.

WewLads replied:
Well what sort of things? I've tried it with all sorts of plants and it works fine.
I mean sure you wouldn't normally smoke a cabbage but there's no harm in it either.

WewLads replied:
I'm looking at the object tree and the only questionable thing that would be included is carpet in plant form.

WewLads committed:
Revert "Fixed the drying rack"

This reverts commit 1351e2f38ed819b734a558fbeb9ef839cddf2b8f.

WewLads committed:
Fixed the drying rack for real this time

This is a very tricky issue as tgstation has the same code but I have no
clue how the hell they're not getting the same error if -1 is supposed
to return false all the time. There was some committ on our end that
caused this issue but I have no clue what it is but this at least fixes
it without runtime errors.
